### PR TITLE
Improve animation examples

### LINKED ISSUE #19852: Missing a 'multiple' example
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/animation

### What specific section or headline is this issue about?

examples

### What information was incorrect, unhelpful, or incomplete?

would be useful to have an example of using the shorthand for multiple animations declared at once.

### What did you expect to see?

more complex example to show real life usage.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/animation`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/animation
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/animation/index.md
* Last commit: https://github.com/mdn/content/commit/3cf6e9bc8ca90917d570a732282242cd704b1dce
* Document last modified: 2022-08-19T08:54:56.000Z

</details>

### PR DESCRIPTION
**Pull request by hanzl035**
### Description

Replaced the existing example in en-US/docs/web/css/animation with five new examples. The five examples are
1. Naive Sun Rise
2. Efficient Sun Rise
3. Animating Multiple Properties
4. Applying Multiple Animations
5. Cascading Multiple Animations

'Naive Sun Rise' is intended as a replacement for the original 'Cyclon Eye' example. 'Cyclone Eye' animates a box model property to create a borg-like loading bar. 'Naive Sun Rise' does the same thing but matches the theme new examples. 'Naive Sun Rise' also contains comments that discourage animating box model properties.

'Efficient Sun Rise' does the same thing as 'Naive Sun Rise' by animating a transform property. There really isn't an explanation, or link to something, explaining what makes the transform property more efficient&mdash;other than a comment about using the GPU.

'Animating Multiple Properties' makes the sun rise and changes the suns color. Two additional properties are changed. One of the new properties is allowed to 'default' instead of being set explicitly in both sides of the 'from/to' clause for demonstration purposes.

'Applying Multiple Animations' is a direct response to issue #19852. The color and rise aspects are separated into two different animations. The animations use differing animation properties for demonstration purposes.

'Cascading Multiple Animations' hints at a common mistake made by people new to animations&mdash;at least, I made it a lot. It is tempting to create two animations that utilize the transform property (expecting that they will combine). Combination does not happen, instead only the properties from the animation lowest in the cascade will be honored.

### Motivation

To resolve issue #19852 and give actionable animation examples.

### Related issues and pull requests

Fixes #19852

### CODE REVIEW

**Review by estelle — COMMENTED** [review]
This looks really good. three suggestions:
1) Add one or two sentences before each example putting in plain English what the example demonstrates.
2) Code format: Your tabs seem half as wide as they should be
3) I suggest getting rid of the 'naive' example. the performant one is the same, but performant, so better. You can add a note about not animating box model properties or properties that cause a layout shift.

**Comment by hanzl035** [comment]
I've tried to incorporate the suggestions from https://github.com/mdn/content/pull/20732#pullrequestreview-1108381174, they were really good!

File: files/en-us/web/css/animation/index.md [hunk 1]
```diff
@@ -97,55 +97,194 @@ Consider providing a mechanism for pausing or disabling animation, as well as us
 
 ## Examples
 
-### Cylon Eye
+> **Note:** Animating [CSS Box Model](/en-US/docs/Web/CSS/CSS_Box_Model) properties is discouraged. Animating any box model property is inherently CPU intensive; consider animating the [transform](/en-US/docs/Web/CSS/transform) property instead.
+
+### Sun Rise
+
+Here we animate a a yellow circle representing the sun on a lightblue background representing the sky. The sun rises
```
Comment (estelle) [inline files/en-us/web/css/animation/index.md:104]:
```suggestion
Here we animate a yellow sun across a light blue sky. The sun rises
```

File: files/en-us/web/css/animation/index.md [hunk 2]
```diff
@@ -97,55 +97,194 @@ Consider providing a mechanism for pausing or disabling animation, as well as us
...
+    transform: translateY(0);
+  }
+}
+```
+
+{{EmbedLiveSample('Sun_Rise')}}
+
+### Animating Multiple Properties
+
+Adding onto the sun animation in the previous example, we add a second animation changing the color of the sun as it rises and sets. The sun starts as a dark red below the horizon changing to a bright orange color as it reaches the top.
```
Comment (estelle) [inline files/en-us/web/css/animation/index.md:143]:
```suggestion
Adding onto the sun animation in the previous example, we add a second animation changing the color of the sun as it rises and sets. The sun starts off dark red when it is below the horizon and changes to a bright orange as it reaches the top.
```

**Review by estelle — APPROVED** [review]
Looks and works fantastic. Thank you! And congratulations on your first MDN PR!
